src/pages/docs/_meta.json

```
@@ -5,6 +5,8 @@
   "w3up-client": "JS Client",
   "how-to": "How to",
   "concepts": "Concepts",
+  "faq": "Frequently Asked Questions",
```

FAQ is fine for navbar. and we can put the long hand as the page h1.
```suggestion
  "faq": "FAQ",
```

---

yeah, it is H1 long hand in the page :)